_includes/contact_form.html

```diff
@@ -168,15 +168,16 @@
     document.addEventListener('DOMContentLoaded', doNotAnswer);
 </script>
 
-<!--Banner for TTP - Remove when ready-->
+<!--Alert Banner - Remove comment out to add or add comment out to remove
```

Comment:
Since we have the git history, we can just delete the whole thing and bring it back if we need to right?

Reply:
Good point. Let's delete it

Reply:
It's deleted.

Reply:
Thanks! ❤️